consensus/consensus-types/src/executed_block.rs

```diff
@@ -106,7 +106,7 @@ impl ExecutedBlock {
 
     pub fn transactions_to_commit(&self) -> Vec<Transaction> {
         // reconfiguration suffix don't execute
-        if self.quorum_cert().ends_epoch() {
+        if self.quorum_cert().certified_block().has_reconfiguration() {
```

Comment:
Lol didn't you just write this in the other PR

Reply:
sadly yes I need to figure out how to cover this in test